### PR TITLE
Abstract apu

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

**Review by tasodorff — CHANGES_REQUESTED** [review]
These all implement AHU, not APU (which isnt defined in the ontology or BDNS/BOS abbreviation standards doc).

**Comment by RathishKJ** [comment]
> These all implement AHU, not APU (which isnt defined in the ontology or BDNS/BOS abbreviation standards doc). Also, these canonical types are all incomplete. They need to be fixed.

The canonicals have been completed with inclusion of new telemetry firlds and abstracts. This has been implemented under MAU.

**Comment by RathishKJ** [comment]
> > These all implement AHU, not APU (which isnt defined in the ontology or BDNS/BOS abbreviation standards doc). Also, these canonical types are all incomplete. They need to be fixed.
> 
> The canonicals have been completed with inclusion of new telemetry fields and abstracts. This has been implemented under MAU.

The Airpurifier Unit is implemented under MAU with new abstract DFVSC3X catering to 3 fans coming in a single Air purifier unit. The air is discharged to the AHU rooms from the APU units

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/APU.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+
+
+APU_SS:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/APU.yaml]: APU.yaml has been removed as an entity. All the associated canonicals comprising of APU has been deleted.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -4966,4 +4966,123 @@ EFHLC:
...
+  description: "Fan run duration monitoring."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - fan_run_time_accumulator_1
+  - fan_run_time_accumulator_2
+  - fan_run_time_accumulator_3
+  
+MSM:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: deleted the Abstract...

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -4966,4 +4966,123 @@ EFHLC:
...
+  implements:
+  - MONITORING
+  uses:
+  - supply_fan_speed_percentage_sensor
+  - low_limit_speed_percentage_command
+  - high_limit_speed_percentage_command
+  - high_speed_status
+  - low_speed_status
+
+MSM3X:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: deleted the Abstract MSM3X...

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -4966,4 +4966,123 @@ EFHLC:
...
+  implements:
+  - MONITORING
+  uses:
+  - dc_voltage_sensor_1
+  - dc_voltage_sensor_2
+  - dc_voltage_sensor_3  
+  - ac_voltage_sensor_1
+  - ac_voltage_sensor_2
+  - ac_voltage_sensor_3
+  - current_sensor_1
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: deleted the abstract...

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 5]
```diff
@@ -752,4 +752,16 @@ MAU_US_SBO_1000_1:
...
+#New canonical#
+MAU_FDPM_SSPC_STC_SHM_DFVSC3X:
+  description: "Air purifier unit (MAU) with 3 fan with differential pressure, static pressure, temperature, humidity and electrical parameter monitoring. "
+  implements:
+  - MAU
+  - FDPM
+  - SSPC
+  - STC
+  - SHM
+  - DFVSC3X
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:766]: This has supply fan and therefore, the DFVSC3X changed to SFVSC3X and SFSS3X has been added to the canonical.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 6]
```diff
@@ -4966,4 +4966,123 @@ EFHLC:
...
+  description: "Fan run mode monitoring."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - fan_run_mode_1  
+  - fan_run_mode_2
+  - fan_run_mode_3  
+  
+FRDM3X:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Deleted the abstract...

**Review by tasodorff — APPROVED** [review]

**Pull request merged**